catalogue_pipeline/transformer/src/main/scala/uk/ac/wellcome/transformer/transformers/sierra/SierraDescription.scala

```diff
@@ -0,0 +1,29 @@
+package uk.ac.wellcome.transformer.transformers.sierra
+
+import uk.ac.wellcome.models.Agent
+import uk.ac.wellcome.transformer.source.{MarcSubfield, SierraBibData}
+
+trait SierraDescription {
+
+  def getDescription(bibData: SierraBibData): Option[String] = {
+    val matchingFields = bibData.varFields
+      .filter {
+        _.marcTag.contains("520")
+      }
+
+    val matchingSubfields =
+      matchingFields.flatMap {
+        _.subfields
+      }.flatten
+
+    matchingSubfields
+      .filter {
+        _.tag == "a"
+      }
+      .map { subfield =>
+        subfield.content
+      }
+      .headOption
```

Comment:
I haven't checked, but it sounds like you're taking the first instance subfield $a.

Some fields/subfields can be repeated; others can't. It would be worth double-checking with Branwen what happens here. Can you have multiple 250 fields? Or subfield $a's on a particular 250?